app/src/main/java/grinnell/appdev/edu/lyles/JSONConstants.java

```diff
@@ -0,0 +1,26 @@
+package grinnell.appdev.edu.lyles;
+
+/**
+ * A class to manage constants related to interaction with the JSON files
+ *
+ * @author Shelby Frazier
+ */
+
+public class JSONConstants {
```

Comment:
Good! As a matter of fact, we can just name this class as `Constants` and put all our constants here. In this class, we'll categorically group constants under an inline comment that tells us what is is. eg:
``` java
public class Constants {
 // URL Constants
public static final String HOT_FOOD_URL = "http://www.cs.grinnell.edu/~birnbaum/appdev/lyles/hotfood.json";
public static final String SNACKS_URL = "http://www.cs.grinnell.edu/~birnbaum/appdev/lyles/snacks.json";

// Array key constants 
public static final String HOT_FOOD_ARRAY_KEY = "hotfood";
public static final String SNACKS_ARRAY_KEY = "snacks";

// Animation constants

// And so on and so forth.
}
```